composer/yamls/recipes/resnet50_medium.yaml

```diff
@@ -0,0 +1,94 @@
+algorithms:
+  blurpool:
+    blur_first: true
+    min_channels: 16
+    replace_convs: true
+    replace_maxpools: true
+  channels_last: {}
+  ema:
+    half_life: 100ba
+    train_with_ema_weights: false
+    update_interval: 20ba
+  label_smoothing:
+    smoothing: 0.1
+  mixup:
+    alpha: 0.2
+    interpolate_loss: false
+  progressive_resizing:
+    delay_fraction: 0.4
+    finetune_fraction: 0.2
+    initial_scale: 0.5
+    mode: resize
+    resize_targets: false
+    size_increment: 4
+  sam:
+    epsilon: 1.0e-12
+    interval: 10
+    rho: 0.5
+callbacks:
+  lr_monitor: {}
+  speed_monitor:
+    window_size: 100
+dataloader:
+  num_workers: 8
+  persistent_workers: true
+  pin_memory: true
+  prefetch_factor: 2
+  timeout: 0.0
+device:
+  gpu: {}
+eval_batch_size: 2048
+eval_interval: 1
+loggers:
+  progress_bar:
+    console_log_level: EPOCH
+    stream: stderr
+model:
+  resnet:
+    initializers:
+    - KAIMING_NORMAL
+    - BN_UNIFORM
+    - LINEAR_LOG_CONSTANT_BIAS
+    loss_name: binary_cross_entropy_with_logits
+    model_name: resnet50
+optimizer:
+  decoupled_sgdw:
+    dampening: 0.0
+    lr: 2.048
+    momentum: 0.875
+    nesterov: false
+    weight_decay: 0.0005
+precision: AMP
+max_duration: 90ep
+scale_schedule_ratio: # Fraction of 90 epochs to train for. 0.75-2.6 for medium recipe
```

Comment:
wait so why don't we set a value here? do we want this to fail by default unless a user explicitly edits the yaml? we want these to work out of the box right?